### DISCUSSION TITLE
"Latest" GitHub release is for a v1 version instead of v2

### DISCUSSION THREAD

**Discussion by samford** [werf/werf#6325] [General]
Hi, I'm a [Homebrew](https://github.com/Homebrew) maintainer and we were recently updating our package for werf to 2.10.5 (https://github.com/Homebrew/homebrew-core/pull/182966) but I noticed that the release that's marked as "Latest" on GitHub is v1.2.328 instead.

We've been using v2 releases and we check the "Latest" release on GitHub to identify the newest version, so I wanted to clarify that 2.10.5 is the latest stable version. If so, could you please mark it as the "Latest" release? This would save us from having to check all recent releases to identify the newest v2 release (instead of only having to check the "latest" release). Thanks!

**Comment by alexey-igrychev** [comment]
We have marked the release as the latest, but I don't know how to automate this for future releases.
